### ISSUE TITLE
Drag to editor to open file [feature request] 

### ISSUE THREAD

**Issue by kandros** [atom/tree-view#638]
It would be usefull to be able to drag a tab in the editor to open it in a new tab

This could even lead to a feature like create split based on where the file is dropped

**Comment by billybonks** [comment]
i dont quite understand what you are saying? drag a editor tab to open a new tab where ?

**Comment by kandros** [comment]
@billybonks edited

**Comment by billybonks** [comment]
ok i understand now, why not just click the file or use a hotkey to open it in another pane like `cmd-k left`.

**Comment by jsejcksn** [comment]
@billybonks Not all users are equally keyboard-focused. Some users are more casual coders and adding this drag-and-drop behavior will make Atom even more accessible. It is a nice enhancement request IMO and consistent with the behavior of many apps especially on OS X.

**Comment by doubleloop** [comment]
It would make switching from JetBrains easier :P

**Comment by EternityForest** [comment]
Makes sense, as long as dragging folders doesn't open a tab for each file in the folder even if there is 30 files like it does in some editors.

**Comment by tofof** [comment]
+1
